blueprints/data-solutions/data-platform-minimal/01-landing.tf

```diff
@@ -19,6 +19,7 @@ locals {
     "roles/storage.objectCreator" = [module.land-sa-0.iam_email]
     "roles/storage.objectViewer"  = [module.processing-sa-cmp-0.iam_email]
     "roles/storage.objectAdmin"   = [module.processing-sa-0.iam_email]
+    "roles/bigquery.dataViewer"   = ["serviceAccount:${module.processing-project.service_accounts.robots.dataform}"]
```

Comment:
We need [tog-staging-dt-prc-0@tog-staging-dt-prc.iam.gserviceaccount.com] to be bq admin on landing

Reply:
ok adding

Reply:
but not curated ?

Reply:
Hmm I dont think so as its dataform that will be talking to curated and that has its own service account. Which reminds me Dataform account will need Admin on curated

Reply:
Ignore that dataform only needs editor on curated